vars/skipStage.groovy

```diff
@@ -143,6 +143,12 @@ boolean skip_ftest(String distro, String target_branch) {
         return true
     }
 
+    // When the stage is started manually or via a timer skipping the stage is
+    // controlled by the build parameter and commit pragmas are ignored
+    if (startedByTimer() || startedByUser()) {
```

Comment:
We also have [startedByUpstream()](https://github.com/daos-stack/pipeline-lib/blob/c80ea0162f5f1bda248544dd6f46eae6d8fe50e5/vars/startedByUpstream.groovy) which sets parameters for the downstream test job.  Should it override commit pragmas also?  Upstream test jobs _can_ take a commit pragma to name (a) branch(es) to use instead of the default of `master` and `release/2.2` for the downstream test job so it's conceivable that it might be asked to use a PR that has commit pragmas.

Reply:
I've reworked this fix and its now the changes to `skip_ftest_hw` are modeled after theapproach used in `skip_ftest`.